index.js

```diff
@@ -1,11 +1,30 @@
 // Import express
 const express = require("express");
-//Import data
+
+// Import file system
+const { writeFile } = require("node:fs/promises");
+
+//Import data from JSON to be read in js
```

Comment:
you should maybe clean up imports -> Right click in `Visual Studio -> Organize imports`